cmake/CMakeLists.txt

```diff
@@ -71,11 +71,13 @@ endif()
 
 configure_file(OpenEXRConfig.h.in ${CMAKE_CURRENT_BINARY_DIR}/OpenEXRConfig.h)
 configure_file(OpenEXRConfigInternal.h.in ${CMAKE_CURRENT_BINARY_DIR}/OpenEXRConfigInternal.h)
+configure_file(../src/lib/OpenEXRCore/OpenEXRVersion.h ${CMAKE_CURRENT_BINARY_DIR}/OpenEXRVersion.h COPYONLY)
```

Comment:
this should not be needed (see below)

Reply:
This one is funky ~ the reason it's there is because OpenEXRConfig.h needs to include <OpenEXRVersion.h> and the file isn't visible to Config, in the `${CMAKE_CURRENT_BINARY_DIR}` prior to installation, and so catch-22 style, the library can't be built without this copy. So the `COPYONLY` thing is just to get a copy of the file into the binary dir at build time.